Makefile

```diff
@@ -14,6 +14,11 @@ ZWRAPDIR = zlibWrapper
 TESTDIR  = tests
 FUZZDIR  = $(TESTDIR)/fuzz
 
+# Define some install locations
+LIBDIR   = $(CMAKE_INSTALL_PREFIX)/lib
+MANDIR   = $(CMAKE_INSTALL_PREFIX)/share/man
```

Comment:
My understanding is that `MANDIR` is different depending on Linux / BSD targets.
Here it seems there is no distinction.